src/run.py

```diff
@@ -51,6 +51,11 @@
         if prob(0.10): #25% chance we'll learn more 
           log.info('going to learn')
           learn()
+        
+        if prob(0.10): # 25% chance we'll delete previous comments with negative upvote
```

Comment:
I would like to see the probability lowered to `0.05`. I'm now seeing that `0.10` is even too much for the learning function as well.